LPS-132257 Add command line property for test task

petershin opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

petershin commented:
@arboliveira - Can we do it this way so this feature can be used by other teams too?

petershin commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-4.liferay.com/job/test-portal-source-format/4857/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-4.liferay.com//userContent/jobs/test-portal-source-format/builds/4857/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/cae1e2b6bf6354639a4bfb0ad40b1929a674b683">cae1e2b6bf6354639a4bfb0ad40b1929a674b683</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/petershin/liferay-portal/tree/m-051701">m-051701</a><br/>Branch GIT ID: <a href="https://github.com/petershin/liferay-portal/commit/004e659be38fd037697045adad0875e982838886">004e659be38fd037697045adad0875e982838886</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-4.liferay.com/job/test-portal-source-format/4857/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-source-format/builds/4857/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-4.liferay.com/job/test-portal-source-format/4857/">test-portal-source-format#4857</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-4.liferay.com/userContent/jobs/test-portal-source-format/builds/4857/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/arboliveira/liferay-portal/pull/876">arboliveira#876</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1533258510">[master] ci:test:sf - petershin &gt; arboliveira - PR#876 - 2021-05-17[20:12:25]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-4.liferay.com/job/publish-testray-report/1256/">publish-testray-report#1256</a><br/></div></div>

Commit:
LPS-167730 Add dependencies

Commit:
LPS-167730 Directly reference the corresponding javax.portlet.Portlet as now the alias is registered before portlet activation; also, each alias maps to a unique portlet name.

Commit:
LPS-167730 Remove unused interface PortletAliasRegistration

Commit:
LPS-167730 Replace DS reference with new API to directly register/unregister the alias

Commit:
LPS-167730 Move SiteNavigationMenuPortletKeys to -api, prepare for next

Commit:
LPS-167730 SF, extract the common string

Commit:
LPS-174196 LPS-181947 Add feature flag

https://issues.liferay.com/browse/LPS-181947

Commit:
LPS-182073 Fix visual details on new client-extension type

Commit:
LPS-174196 LPS-181755 portal-search-elasticsearch7-impl: Add searchAfter implementation

https://issues.liferay.com/browse/LPS-181755

dsanz committed:
LPS-177720 effectively hide the internal checkbox to the screen reader and tab order

Commit:
LPS-174421 New Resource Action

Commit:
LPS-174421 Make AuditDisplayContext compatible with ResourceRequests

Commit:
LPS-174421 PortletConfigurationIcon to show export option

Commit:
LPS-174421 ResourceAction for CSV download

Commit:
LPS-174421 Export all events, not just the current page

dsanz committed:
LPS-182073 build lang

vicnate5 committed:
LRQA-80198 Create build-test-local for local testing

vicnate5 committed:
LRQA-80198 Move docker database scripts over to new file since they are only used in local testing

vicnate5 committed:
LRQA-80198 Set default mysql to match CI

Commit:
LPS-183189 Simplify permissions, restrict to admin only.

For LPS-174421.

Commit:
LPS-174421 buildLang

Commit:
LPS-174196 LPS-183090 Create deep pagination configuration

- Point In Time configurable
- Enable Deep Pagination configurable

https://issues.liferay.com/browse/LPS-183090

Commit:
LPS-167730 Apply

CrescenzoRegaLF committed:
COMMERCE-10726 Add new fields and path (yaml files)

CrescenzoRegaLF committed:
COMMERCE-10726 Adapt resource classes to the new logic

CrescenzoRegaLF committed:
COMMERCE-10726 Adapt the dto converter to the new logic

CrescenzoRegaLF committed:
COMMERCE-10726 Adapt the code to the new APIs

CrescenzoRegaLF committed:
COMMERCE-10726 Adapt integration test classes to the new logic

leilanyaragao committed:
LRAC-13557 Enable SF osb-faro

leilanyaragao committed:
LRAC-13557 osb-faro-contacts-demo: change elasticsearch-snapshot version

leilanyaragao committed:
LRAC-13557 osb-faro-mock-engine-client: Update build.gradle

leilanyaragao committed:
LRAC-13557 SF

leilanyaragao committed:
LRAC-13269 osb-faro-engine-client: Update bnd.bnd

Commit:
LPS-174196 portal-language-lang: Add language key

https://issues.liferay.com/browse/LPS-174196

Commit:
LPS-174196 LPS-183090 hide UI

vicnate5 committed:
LRQA-80198 Move cluster setup since it is only used locally

vicnate5 committed:
LRQA-80198 Create setup for bundles downloaded from ci

vicnate5 committed:
LRQA-80198 Add imports

Commit:
COMMERCE-11398 Use guest account if searching as guest user

Commit:
LPS-174196 LPS-183159 portal-search-elasticsearch7: Add configuration to update max result window setting from elasticsearch

https://issues.liferay.com/browse/LPS-174196

david-truong committed:
LPS-183473 only let recycle be emptied in production

MarinhoFeliphe committed:
LPS-183422 Split ObjectEntryManager interface to make it extensible and better understandable

MarinhoFeliphe committed:
LPS-183422 Rename to updateObjectEntry because addOrUpdate only makes sense for DefaultObjectEntryManagerImpl

MarinhoFeliphe committed:
LPS-183422 Adapt implementations

MarinhoFeliphe committed:
LPS-183422 ObjectDDMStorageAdapter only supports DefaultObjectEntryManager because this adapter is using the methods that are only implemented by DefaultObjectEntryManagerImpl

MarinhoFeliphe committed:
LPS-183422 Only objectEntryManagers that implements ObjectEntryRelatedObjectsManager should be allowed to use thi to call these methods

MarinhoFeliphe committed:
LPS-183422 Only implementations of StandaloneObjectActionManager and ObjectEntryRelatedObjectsManager can use the methods related

MarinhoFeliphe committed:
LPS-183422 DefaultObjectEntryManager is the expected here since the methods used are the ones implemented only by the default object entry manager

MarinhoFeliphe committed:
LPS-183422 Predicate should not be created at this layer since it is not proper to be used for other objectEntryManagers

MarinhoFeliphe committed:
LPS-183422 Maybe we don't need these interfaces for now

MarinhoFeliphe committed:
LPS-183422 Not need

MarinhoFeliphe committed:
LPS-183422 Adapt DefaultObjectEntryManagerImplTest

MarinhoFeliphe committed:
LPS-183422 Semver

david-truong committed:
LPS-183473 if item is in trash show that it is deleted in UI

marcogalluzzi committed:
LPS-182876 Add "Copy To" action in folders context menu under Feature Flag

marcogalluzzi committed:
LPS-182876 Added "Copy between repositories" in DLAppService

marcogalluzzi committed:
LPS-182876 auto buildservice + baseline

marcogalluzzi committed:
LPS-182876 Added Action and Render commands

marcogalluzzi committed:
LPS-182876 Added new page with display context class

marcogalluzzi committed:
LPS-182876 Added new language properties

marcogalluzzi committed:
LPS-182876 auto build language

shuyangzhou committed:
LPS-184594 Let DependencyManagerSync taking over the sync future task thread creation.

shuyangzhou committed:
LPS-184594 Semantic versioning

Commit:
LPS-183732 Updating publication tests do not empty the recycle bin

Commit:
COMMERCE-11395 Add scroll in modal body

Commit:
LPS-183732 Remove the inapplicable tests

Commit:
LPS-174196 gw buildLang

https://issues.liferay.com/browse/LPS-174196

Commit:
COMMERCE-11395 fix prettier error

marcogalluzzi committed:
LPS-182876 minor improvements

marcogalluzzi committed:
LPS-182876 Added icon to "Copy To" folder action in context menu

marcogalluzzi committed:
LPS-182876 Moved "Copy To" folder action after the "Move" action

kildarep committed:
LPS-182988 Removing the kebab when creating or editing something in Testray projects

david-truong committed:
LPS-183473 don't allow empty recycle bin in quick actions when in a publication

rafaprax committed:
LPS-183802 Patch nimbus-jose-jwt

rafaprax committed:
LPS-183802 Create module to hold the outlook connector implementation

jaydawu committed:
COMMERCE-10603 Add and update macros

Commit:
LPS-183985 Cannot empty recycle bin within publication

Commit:
LPS-183732 Workaround the steps to view changes

jaydawu committed:
COMMERCE-10603 Add test CanViewProductsWithCategoryImportedFromAnotherInstanceViaJSONAdd

ricardocousolr committed:
LPS-184014 Take possible non ROOT context into account

leilanyaragao committed:
LRAC-13557 skip JavaReferenceAnnotationsCheck

rafaprax committed:
LPS-183802 Create the MailAuthTokenProvider API to enable the implementation of specific authentication flow for each mail server provider. For now, we'll add the connector for the outlook but there are plans to add for Gmail as well

rafaprax committed:
LPS-183802 Baseline

markocikos committed:
LPS-183656 Build lang

Commit:
LPS-174196 LPS-183159 portal-search-elasticsearch-impl: Fix CompanyIndexFactoryTest class

https://issues.liferay.com/browse/LPS-174196

markocikos committed:
LPS-183656 Implement editing of FDS Fields

jorgediaz-lr committed:
LPS-184158 Add falback when the DEDataDefinitionFieldLink is not found using UUID+groupId because we have remote staging with two upgrades in parallel, so the upgrade added by LPS-103896 generated different UUIDs in each environment, but the IDs matches

rafaprax committed:
LPS-183802 Lang key

gamerson committed:
LPS-181325 fix assemble

nNilton committed:
LPS-184211 add information templates

MarinhoFeliphe committed:
LPS-183422 Create DefaultObjectEntryManagerProvider and apply to use

ces-quanhuynh committed:
LPS-183757 Center publication modal entry titles

markocikos committed:
LPS-183656 Rename label

markocikos committed:
LPS-183656 Deduplicate

markocikos committed:
LPS-183656 Define cell render labels locally

marcosapmf committed:
LRAC-13557 fix gradle packageRunWebpack task

Commit:
LPS-184197 Add Preview URL to metadata for attachment fields

Commit:
LPS-184197 Add Preview URL to language.properties

Commit:
LPS-184197 buildlang

marcogalluzzi committed:
LPS-182876 Added tests for the new folder Copy method

lorenzolapescara committed:
COMMERCE-11398 Add test CanB2XGuestUsersSearchProducts

drewbrokke committed:
LPS-147105 gradle-plugins-workspace: uses copySpec.forEach on the files to rename the paths, then clean up the unversioned tomcat directory

drewbrokke committed:
LPS-147105 gradle-plugins-workspace: updates initBundle test case

drewbrokke committed:
LPS-147105 gradle-plugins-workspace: updates distBundleZip test case

Commit:
LPS-183189 Follow pattern from the Instance Settings portlet to restore admin only portlet access and no assignable permissions.

For LPS-174421.

noornajj committed:
LPS-184344 Change the model listner to onAfterUpdate to avoid StaleStateException

noornajj committed:
LPS-184344 Pass in the original layout to isSkipEvent instead of fetching it

nNilton committed:
LPS-184211 add Test to assert information templates

Commit:
LPS-181022 Feature Flag.

For LPS-174421.

drewbrokke committed:
LPS-147105 gradle-plugins-workspace: updates test cases to assert that the unversioned tomcat directory is removed from the destination

melodywu committed:
LPS-182413 Add locators for autdit export csv

melodywu committed:
LPS-182413 Add macro for autdit csv

melodywu committed:
LPS-182413 Add testcase for autdit csv

thektan committed:
LPS-178738 LPS-184188 Add delete action to Fields tab

thektan committed:
LPS-178738 LPS-184188 New language key

thektan committed:
LPS-178738 LPS-184188 gradlew buildLang

Commit:
LPS-184140 Add test to edit and adapt image resolution in a publication

Commit:
LPS-184141 Add test to disable and delete image resolution in publication

thektan committed:
LPS-182790 LPS-184492 Match custom text

thektan committed:
LPS-182790 LPS-184492 Add fuzzy highlighting

thektan committed:
LPS-182790 LPS-184492 yarn types

Commit:
LPS-183732 Update the conflict message

Hanlf committed:
LPS-180322 Update macros to include fileName and title vars

Hanlf committed:
LPS-180322 Add macros about document

allen-nagtalon committed:
LRQA-80461 Replace deprecated runScript method

Commit:
LPS-184138 Add test to add and adapt image resolution in a publication

Commit:
LPS-182073 Fix language key

nNilton committed:
LPS-184211 simplify

nNilton committed:
LPS-184211 add update tests

nNilton committed:
LPS-184211 sort

naraandrade committed:
LPS-178499 Fix update of currentRecipients

nNilton committed:
LPS-184211 rename ERC

nNilton committed:
LPS-184211 make consistent

rafaelfs98 committed:
LPS-183685 The case parameters do not remain selected when editing

ling-alan-huang committed:
LPS-184438 Move to proper package

ling-alan-huang committed:
LPS-184438 Move to proper package

slnn committed:
LPS-180567 Remove the method reference from EntityCacheImpl by using Snapshot

slnn committed:
LPS-180576 Update test

joseabelenda committed:
LPS-184211 Simplify

joseabelenda committed:
LPS-184211 Auto SF

Commit:
LPS-174196 portal-language-lang: Edit language key

https://issues.liferay.com/browse/LPS-174196

Commit:
LPS-174196 gw buildLang

https://issues.liferay.com/browse/LPS-174196

Commit:
LPS-174196 portal-search-elasticsearch7: Remove random Blank lines

https://issues.liferay.com/browse/LPS-174196

allen-nagtalon committed:
LRQA-80463 Replace deprecated typeCKEditor method

rafaelfs98 committed:
LPS-184466 Disable autofill buttons

welitonpe committed:
LPS-184426 Fixed permissions to accessn create page

welitonpe committed:
LPS-184426 fixed request to caseform and requirementsForm

aline-doleron committed:
LPS-183270 Create component to generate avatars

aline-doleron committed:
LPS-183270 Use gravatar api to generate members placeholder images

aline-doleron committed:
LPS-183270 Add dev dependency

marcosapmf committed:
LRAC-13557 use reference annotation the the variable instead of setter method

marcosapmf committed:
LRAC-13557 fix packageRunFormat

MarinhoFeliphe committed:
LPS-183422 Define DefaultObjectEntryManager as a paramenter instead of ObjectEntryManager

MarinhoFeliphe committed:
LPS-183422 Use DefaultObjectEntryManagerProvider

thektan committed:
LPS-178738 LPS-184188 Refactor - Use async await

vicnate5 committed:
LRQA-80198 Persist app.server.parent.dir when using run-selenium-tomcat for local users who are not using the default location

vicnate5 committed:
LRQA-80198 Keep database logic clean and traceable

vicnate5 committed:
LRQA-80198 Resolve issue caused by mismatch between ci directory name and custom app.server.parent.dir

vicnate5 committed:
LRQA-80198 SF

ling-alan-huang committed:
LPS-184495 Do not add modified method with @Modified annotation if it only calls deactivate and then activate

ling-alan-huang committed:
LPS-184495 Only enable for master for now

brianchandotcom committed:
LPS-182073 Wordsmith

brianchandotcom committed:
LPS-182073 Regen

brianchandotcom committed:
LPS-174421, LPS-182413 Rename

brianchandotcom committed:
LPS-174421, LPS-182413 Rename

brianchandotcom committed:
LPS-174421, LPS-182413 As used

brianchandotcom committed:
LPS-174421, LPS-182413 auto SF

CrescenzoRegaLF committed:
COMMERCE-10726 Autogenerated code (blade gw buildService)

CrescenzoRegaLF committed:
COMMERCE-10726 Autogenerated code (blade gw buildREST)

CrescenzoRegaLF committed:
COMMERCE-10726 Semantic versioning (gw baseline)

ling-alan-huang committed:
LPS-184495 Generated

ling-alan-huang committed:
LPS-175368 Need to check *.jsx for generating FeatureFlags

Commit:
LPS-184501 Increase timout explicit wait to make case stable

brianchandotcom committed:
LPS-183802 Wordsmith

brianchandotcom committed:
LPS-183802 Inline

brianchandotcom committed:
LPS-183802 prep next

brianchandotcom committed:
LPS-183802 Wordsmith

brianchandotcom committed:
LPS-183802 Regen

Commit:
LPS-184502 Modify macro to make it reusable

Commit:
LPS-184502 Add missing parameter

Commit:
LPS-184503 Modify case to adapt new behavior

brianchandotcom committed:
LPS-157718 Typo

JessieZhao0714 committed:
LPS-184159 Update locator

Hanlf committed:
LPS-180322 Add and update tests about Document

Hanlf committed:
LPS-180322 Update macro to make it work for site document

Hanlf committed:
LPS-180322 Assert document is not added via API

gaborlovasdotcom committed:
LRQA-80627 Update XPath

bryceosterhaus committed:
LPS-183719 - Add lint rule and apply for TS interface naming

Commit:
LPS-167730 Semantic versioning

gaborlovasdotcom committed:
LRQA-78829 Adding OSGi configuration

bryceosterhaus committed:
LPS-183719 - regen types

JoyceWang08211 committed:
LPS-183857 Update test

Commit:
LPS-184527 Update the number of adapted images

liferay-translation committed:
LPS-77699 Update Translations

joelrocha68 committed:
LPS-184529 Fix teardown

Commit:
LPS-181548 talend - simplify regex used for input field validation in Talend Open Studio (reported for ReDoS)

marcosapmf committed:
LRAC-13557 SF

CrescenzoRegaLF committed:
COMMERCE-10726 Rename

brianchandotcom committed:
saml-web 5.0.59 prep next

Commit:
LPS-184541 - Items need to be added to a dropdown item group.

Commit:
LPS-179937 Fix MDF Claim had no permission to comment

thaynam committed:
LPS-184479 Add save as draft alert to mdf forms

naraandrade committed:
LPS-183270 Show placeholder icon only if member does not have an uploaded image

petershin committed:
LPS-147105 Variable name

petershin committed:
LPS-147105 Inline

petershin committed:
LPS-147105 Inline

gaborlovasdotcom committed:
LRQA-80644 Temporary ignore test case

thaynam committed:
LPS-184479 Fix Deal Reg name in Tab

thaynam committed:
LPS-184479 Add dialect tab to mdf request and claim pages

thaynam committed:
LPS-184479 Make back button redirect to mdf claim list

thaynam committed:
LPS-184479 Remove unused fragments

rafaellajordao committed:
LPS-173652 create CategoriesFragment.testcase

rafaellajordao committed:
LPS-173652 add vocabularyType to PageEditor.configureFragment

rafaellajordao committed:
LPS-173652 add openSelectCategory macro

rafaellajordao committed:
LPS-173652 add MapCategoriesFragmentToCategoriesFieldInMasterPage test

rafaellajordao committed:
LPS-173652 add MapCategoriesFragmentToCategoriesFieldInPageTemplate test

Commit:
LPS-178427 Add component name to test suite

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/dxp/apps/osb/osb-faro/.gitrepo'.

shuyangzhou committed:
LPS-184626 Component is going to be deactivated anyway, no need to clear.

shuyangzhou committed:
LPS-184626 Not used anywhere.

shuyangzhou committed:
LPS-184626 Proper bundle tracking

shuyangzhou committed:
LPS-184626 Private fields.

raquelpaiva committed:
LPS-184482 Add breadcrumb in All News page

shuyangzhou committed:
LPS-184626 Lazy opening BundleTracker in FrontendTokenDefinitionRegistryImpl

rafaellajordao committed:
LPS-173652 add ViewAllCategoriesAvailableToSiteScopeObject test

thektan committed:
LPS-178738 LPS-184188 Fix interface names

rafaellajordao committed:
LPS-173652 add ViewOnlyGlobalVocabulariesAvailableToCompanyScopeObject test

rafaellajordao committed:
LPS-173652 add ViewWarningMessageWithCategorizationDisabled test

allen-ziegenfus committed:
LCD-26596 Add missing portlet category names

thaynam committed:
LPS-184478 Fix contact info changing when edit mdf request

Commit:
LPS-173652 Use general locator

Commit:
LPS-184629 Remove deprecated property

Commit:
LPS-184632 Fix typo

Commit:
LRQA-80640 Fix typo

Commit:
LPS-184631 Remove redundant step

brianchandotcom committed:
LPS-178738 Wordsmith

brianchandotcom committed:
LPS-178738 Regen

Commit:
LPS-184631 Modify cases to adapt new behavior

brianchandotcom committed:
LPS-147105 prep next

Commit:
LPS-184635 Increase timout explicit wait to make case stable

brianchandotcom committed:
LPS-174196 Wordsmith

brianchandotcom committed:
LPS-174196 regen

brianchandotcom committed:
Auto SF

brianchandotcom committed:
OSB Customer Portal Upload

Commit:
LPS-184658 Modify case to adapt new behavior

brianchandotcom committed:
LPS-182876 Wordsmith

brianchandotcom committed:
LPS-182876 not needed

liferay-translation committed:
LPS-77699 Update Translations

liferay-translation committed:
LPS-77699 Update Translations

petershin committed:
LPS-148692 Partial revert
